### PR TITLE
Moving changelog endpoint

### PR DESCRIPTION
**Pull request by dolgopolovwork**
Changelog code goes to a new repository.

### CODE REVIEW

**Review by yurivin — COMMENTED** [review]
So, does swagger supported? It is not about that service has only one endpoint. It is about dozen of services that have hundreds of endpoints

**Comment by dolgopolovwork** [comment]
> So, does swagger supported? It is not about that service has only one endpoint. It is about dozen of services that have hundreds of endpoints

Not yet. I guess a new task will be created for that.

File: changelog-endpoint/src/integration-test/kotlin/endpoint/ChangelogModuleIntegrationTest.kt [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+        with(handleRequest(HttpMethod.Post, "/changelog/changelogFile") {
+            setBody(changelogEnvironment.gson.toJson(request))
+        }) {
+            assertEquals(HttpStatusCode.OK, response.status())
+        }
+        //Second changelog attempt
+        with(handleRequest(HttpMethod.Post, "/changelog/changelogFile") {
+            setBody(changelogEnvironment.gson.toJson(request))
+        }) {
+            assertEquals(HttpStatusCode.OK, response.status())
```
Comment (Mingela) [inline changelog-endpoint/src/integration-test/kotlin/endpoint/ChangelogModuleIntegrationTest.kt:97]: status is ok but how can we guarantee that nothing changed?

**Review by Mingela — APPROVED** [review]

File: changelog-endpoint/src/integration-test/kotlin/endpoint/ChangelogModuleIntegrationTest.kt (hunk 1 above)
Comment (dolgopolovwork) [reply changelog-endpoint/src/integration-test/kotlin/endpoint/ChangelogModuleIntegrationTest.kt:97]: It's a good point. I will modify the test.

**Pull request merged**